packages/target-electron/src/deltachat/webxdc.ts

```diff
@@ -106,7 +126,8 @@ export default class DCWebxdc {
     const openWebxdc = async (
       _ev: IpcMainInvokeEvent,
       msg_id: number,
-      p: DcOpenWebxdcParameters
+      p: DcOpenWebxdcParameters,
+      is_maps_xdc = false // special behaviour for the map dc integration (in this case bigger landscape window)
```

Comment:
Here we could have a `defaultBounds` parameter instead. This method shouldn't care if we are running a map or not.